spec/unit/util/package/version/debian_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+require 'puppet/util/package/version/debian'
+
+describe Puppet::Util::Package::Version::Debian do
+  context "when creating new version should fail" do
+    it "if is parsing symbols" do
+      expect { Puppet::Util::Package::Version::Debian.parse(:absent) }.to raise_error(Puppet::Util::Package::Version::Debian::ValidationFailure)
```

Comment:
`described_class` can be used instead of the exact, but long, class name `Puppet::Util::Package::Version::Debian`